src/integrations/feed/clients.ts

```diff
@@ -16,9 +17,20 @@ type FeedFetchOptions = RequestInit & {
 /**
  * Naive implementation of a feed client that fetches the feed from the feed service
  */
-export class FeedClient implements IFeedClient {
+export class FeedClient implements IFeedClient, IGarmrClient {
   private readonly url: string;
   private readonly fetchOptions: FeedFetchOptions;
+  private static readonly garmr = new GarmrService({
+    breakerOpts: {
+      halfOpenAfter: 3 * 1000,
+      threshold: 0.1,
+      duration: 30 * 1000,
```

Comment:
defaults in line what platform projects do, [see](https://dailydotdev.slack.com/archives/C07KEKPVBJ9/p1725281135140569?thread_ts=1725280854.868329&cid=C07KEKPVBJ9), we can tune them as we see fit TBD

halfOpenAfter - time after which circuit goes to half open state and tests service health 
threshold - percentage of requests that need to fail for breaker to open
duration - rolling window in which threshold is counted